Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "dapr"
-version = "0.14.0"
+version = "0.14.1"
```

Comment:
As far as I understand the release process follows the one from the main repo, so I don't think we should upgrade it here.

Reply:
Okay. I wasn't sure what the version management rules were. I can revert this change to the version.

Reply:
tbh I'm learning as well. I suppose that's how it works even if I saw @danielgerlag's PR changing it. I would say we don't know in which version it will go so a PR shouldn't change it, especially you could have x PRs in parallel.

Reply:
We can accept this for the upcoming release

Reply:
@cscetbon please revert to 0.14.0

Reply:
@yaron2 @cscetbon Reverted to 0.14.0

Reply:
> @cscetbon please revert to 0.14.0

Not mine though 😬